canonical-websites/snapcraft.io
# Verified publishers

# Done

Fixes: https://github.com/canonical-websites/snapcraft.io/issues/696
Implement verified publisher icon (as described in https://github.com/CanonicalLtd/snapcraft-design/issues/464) on search results and snap details pages.

The API is not yet implemented, but when it is this should 'just work'.

# QA

- Pull this branch
- `./run`
- Remove the following lines:
  - `templates/search.html:59`
  - `templates/search.html:64`
  - `templates/snap-details.html:58`
  - `templates/snap-details.html:63`
- Check the designs

# Screenshots

Note: Shows for all publishers.

![screenshot_2018-05-29 snap search results for 2](https://user-images.githubusercontent.com/479384/40667274-ae7b6a88-6359-11e8-9c53-fd27eaecd542.png)
![screenshot_2018-05-29 install vlc for linux linux apps in seconds snap store](https://user-images.githubusercontent.com/479384/40667278-b2920280-6359-11e8-8f72-9bc381b75140.png)
![screenshot from 2018-05-29 16-03-22](https://user-images.githubusercontent.com/479384/40667374-f26a826a-6359-11e8-977b-bb2cfb9e2b13.png)
